my-webxr-app/src/types/DataPointTypes.tsx

```diff
@@ -0,0 +1,17 @@
+import { SphereGeometryProps } from '@react-three/fiber';
+
+/**
+ * Define an interface to require an ID number to differentiate each DataPoint
+ * and allow other mesh properties to be set.
+ */
+export interface DataPointProps {
+  id: number;
+  columnX: string;
+  columnY: string;
+  columnZ: number;
+  color: string
+  marker: string
+  outlineScale?: number;
+  size?: SphereGeometryProps['args'];
+  meshProps?: JSX.IntrinsicElements['mesh'];
+}
```

Comment:
I do like this change, but if the columns are meant to be possible columns of data to display, acting like all the data a point can have, we may want to make it an array/map so we can have more then just 3 pieces of data held within a datapoint. my understanding of this may be wrong though and this is good for the current sprint.

Reply:
Yes, good point! @trangnguyen3010 mentioned that with the current scope of the project, the number of chosen columns would just be 3. 

But since the DataPointProps interface is a separate directory, we can easily change the structure into an array or map as needed for future sprints.